### ISSUE TITLE
Inconsistency when creating/applying a disguise via API

### ISSUE THREAD

**Issue by jaylawl** [libraryaddict/LibsDisguises#545]
```
        PlayerDisguise disguise = new PlayerDisguise("");
        disguise.setSkin(gameProfile);
//        disguise.setSkin("jaylawl");

        disguise.setDisplayedInTab(false);
        disguise.setUpsideDown(true);
        disguise.setNameVisible(false);

        disguise.setEntity(entity);
        disguise.startDisguise();
```
Using this will have the middle section options ignored. The disguise is not upside down and the nameplate remains visible. The skin itself however works fine.
However if i instead use the above commented-out line where it is my username, instead of the game profile string, everything works just as expected.

Shuffling the options around a bit sometimes makes some of them work. For example, the disguise *will* be upside down, if i apply that option *after* starting the disguise (for some reason that also gives it the name Dinnerbone again). But all that doesn't feel like intentional behaviour.

Here is one of the game profile strings i am using:
```
{"id":"c963565c-2e88-4f3d-8c3b-0d8da1c6303e","name":"Unknown","properties":[{"name":"textures","value":"ewogICJ0aW1lc3RhbXAiIDogMTYxMTc1ODEyMzkzMCwKICAicHJvZmlsZUlkIiA6ICJlZDUzZGQ4MTRmOWQ0YTNjYjRlYjY1MWRjYmE3N2U2NiIsCiAgInByb2ZpbGVOYW1lIiA6ICI0MTQxNDE0MWgiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMTk5ODllOWMwMzMzMGI3OTNlNDFjZTQ5YWM0YTgyZTc5MGFhMzdlMGZlNjY2MzI3NGVhYjhkMmI1MjAzOGFlMyIKICAgIH0KICB9Cn0=","signature":"aX1wmVv/yKFGm1GouRb0qLVGGoqcC+zBVwp/jO0MgC0ozSuhRKccS19b7+UxEg/+To30DoW0ZR/rBJAgfttZV6Nvo2I7dFi7NuxdUS2fkEOcUOAsdFQVp2VjVLkqE2BmpbKs8Mf2666aho6ZWxLXX46XX7dLV8dZYwhCfiOk0jvgrpabR2mhG9r7adkRDrYoikhwLa28pSKpA/VcuQ2uA/EKN/GTOfKMNbSx7+cnOBZBO4a/pGEre9lHYMSKzjTWIR4tqwEBP+AUrw4tKYB7uphzlXkoV5y3ImG/qE3BfwoxmJhlnGt0MJIUGd9IRvTi0RiXifk2fcp3q03Yx0ez+DirggV2JJqA5C2gY22KsB0c97q5xmnAgJgd7lHEWtcgNe6+s7pAU0vjPs/1qFVrKwOG6f3D+dLPn/rgnRrZYrM4P5l1xR3eIyDGTgj11RCFws7/2NM5GIild2M1oGoykthnKsTaRR3uZeyJimn8IMPvuRPqUQvnUu71nnRSDAdeIQGekcaUVdkig7Gcu7dzonXCrti3Gcsgt/gRpturLklBPHUrQF99z5dFCbTyXrZdT7kt+54XTzyCxB2q0/NMy/tpdAlXogHGsiGmTJpuwNz9h8bLRkDTx6hbk1iQtRoU6FS7clvXwANSfAbKHM4L6CbJP78QcSZA12aFNrC0R5E="}],"legacy":false}
```

tl;dr Sometimes things work and sometimes they don't, depending on how you shuffle them around. What i did in that code snippet above used to work perfectly fine back in ~early 2020.

Using Paper 1.16.4 on Java 11

**Comment by libraryaddict** [comment]
Fixed in latest build

**Comment by jaylawl** [comment]
> 
> 
> Fixed in latest build

I can confirm, fixed on my end too, thanks!
